plotly/dashboard_objs/dashboard_objs.py

```diff
@@ -0,0 +1,441 @@
+"""
```

Comment:
should we include a 
```
from dashboard_objs import dashboard_objs
```

inside `__init__.py` that way you just have to do:

```
plotly.dashboard_objs.Dashboard
```

instead of 
```
plotly.dashboard_objs.dashboard_objs.Dashboard

```